FormatFactory.py

```diff
@@ -1,27 +1,41 @@
 from GenotypeFormat import GenotypeFormat
 from GencallFormat import GencallFormat
+from ThetaFormat import ThetaFormat
```

Comment:
Are we supporting the ThetaFormat? If not, we should remove this.

Reply:
This should be removed... good catch.